ui/widgets/autocomplete.js

```diff
@@ -267,8 +268,11 @@ $.widget( "ui.autocomplete", {
 				// Announce the value in the liveRegion
 				label = ui.item.attr( "aria-label" ) || item.value;
 				if ( label && String.prototype.trim.call( label ).length ) {
-					this.liveRegion.children().hide();
-					$( "<div>" ).text( label ).appendTo( this.liveRegion );
+					clearTimeout( this.liveRegionTimer );
+					var that = this;
```

Comment:
No need to use an extra variable as `this` is mapped when using `this._delay`. 

Sorry, I should have been more explicit about this in my previous review.